src/bite.v1.js

```diff
@@ -1,6 +1,7 @@
 (function () {
     const state = {
-        programId: null
+        programId: null,
+        sessionId: null,
```

Comment:
This is not yet sent to the backend yet, it's just stored in the client side state